mailpoet/assets/js/src/common/listings/newsletter_status.tsx

```diff
@@ -3,6 +3,10 @@ import classnames from 'classnames';
 import { addDays, differenceInMinutes, isFuture, isPast } from 'date-fns';
 import { t } from 'common/functions/t';
 import { Tooltip } from '../tooltip/tooltip';
+import {
+  NewsLetter,
+  NewsletterStatus as NewsletterStatusEnum,
+} from '../../newsletters/models';
```

Comment:
This can be replaced as well.
```suggestion
} from 'newsletters/models';
```